java/http/java-http-consumer/src/main/java/HttpConsumer.java

```diff
@@ -171,7 +225,9 @@ public void poll() {
                     log.info("Record {}", record);
                     this.messageReceived++;
                 }
-
+                if (records.size() > 0 && !enableAutoCommit) {
```

Comment:
```suggestion
                if (records.size() > 0 && !config.getEnableAutoCommit();) {
```

Reply:
Good catch Kyle, then `enableAutoCommit` can be removed.

Reply:
Removed